plugins/modules/monitoring/datadog/datadog_monitor.py

```diff
@@ -51,7 +51,7 @@
         description:
           - The type of the monitor.
           - The types C(query alert), C(trace-analytics alert) and C(rum alert) were added in community.general 2.1.0.
-        choices: ['metric alert', 'service check', 'event alert', 'process alert', 'log alert', 'query alert', 'trace-analytics alert', 'rum alert']
+        choices: ['metric alert', 'service check', 'event alert', 'process alert', 'log alert', 'query alert', 'trace-analytics alert', 'rum alert', 'composite']
```

Comment:
You can also remove the `'` signs, since this is a YAML list :) Maybe converting it to a "regular" YAML list (one item per line following `-`) might be best.

Reply:
Thanks, did both.